tests/mesolve/open_system.py

```diff
@@ -83,7 +83,7 @@ def __init__(
         self.H = self.delta * adag @ a
         self.H_batched = [0.5 * self.H, self.H, 2 * self.H]
         self.jump_ops = [torch.sqrt(self.kappa) * a, dq.eye(self.n)]
-        self.exp_ops = [(a + adag) / sqrt(2), 1j * (adag - a) / sqrt(2)]
+        self.exp_ops = [dq.position(self.n), dq.momentum(self.n)]
```

Comment:
Random thought : should be add a `g` argument in `position` and `momentum` (set to 2 by default)?

Reply:
Yes, I think it's a good idea 👍